.eslintrc.js

```diff
@@ -73,12 +75,89 @@ module.exports = {
     ],
     'unicorn/filename-case': ['error', { case: 'kebabCase' }],
     '@typescript-eslint/ban-ts-comment': ['warn'],
-    '@typescript-eslint/no-var-requires': ['warn'],
+    '@typescript-eslint/no-var-requires': ['off'],
```

Comment:
This change seems unrelated, I'd think we have to discuss whether disabling this rule or not.

Reply:
Sure thing. I don't really know why those files are being imported using requires.

FWIW, I turned it off because we didn't previously have this rule until the introduction of the TypeScript eslint package, so it seemed safe to turn off and add back if we found it was causing issues with TypeScript stuff _in particular_. There's apparently no issue with it for other things?

Reply:
Also, based on the description that the ESLint plugin gives, I think we can safely disable it anyway: https://typescript-eslint.io/rules/no-var-requires/#when-not-to-use-it